chall04/isoria-g.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    isoria-g.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: marvin <marvin@student.42.fr>              +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/04/13 21:23:38 by marvin            #+#    #+#              #
+#    Updated: 2020/04/13 21:23:38 by marvin           ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+import sys
+import os
+
+def findnum():
+    totalsize = 0
+    for item in sizes:
+        totalsize = totalsize + item
+    return (totalsize // maxshelf + 1)
```

Comment:
It is a bad idea to use a global variable in a function. It might not be declared